Automattic/wc-calypso-bridge
# Update OBW step headings

Fixes #78 

Replaces the steps with our own steps and adds the step header in place of step navigation.

### Screenshot

<img width="746" alt="screen shot 2018-10-25 at 4 44 40 pm" src="https://user-images.githubusercontent.com/10561050/47532225-63169e00-d875-11e8-9ad7-58a488b962a0.png">
<img width="764" alt="screen shot 2018-10-25 at 4 44 48 pm" src="https://user-images.githubusercontent.com/10561050/47532226-63169e00-d875-11e8-966d-b72258d85b39.png">

### Testing

1.  Visit `/wp-admin/admin.php?page=wc-setup&calypsoify=1`
2.  Check that the heading styles for each step match those in p58i-7kf-p2
